plyer/platforms/linux/orientation.py

```diff
@@ -0,0 +1,23 @@
+from subprocess import call, check_output
+from plyer.facades import Orientation
+
+
+class XrandrOrientation(Orientation):
+
+    def _set_landscape(self, **kwargs):
+        self.rotate = 'normal'
+        self.screen = check_output("xrandr -q | grep ' connected' \
+                              |  head -n 1 | cut -d ' ' -f1", shell=True)
+        self.screen = self.screen.split('\n')[0]
+        call(["xrandr", "--output", self.screen, "--rotate", self.rotate])
+
+    def _set_portrait(self, **kwargs):
+        self.rotate = 'left'
+        self.screen = check_output("xrandr -q | grep ' connected' \
+                              |  head -n 1 | cut -d ' ' -f1", shell=True)
+        self.screen = self.screen.split('\n')[0]
+        call(["xrandr", "--output", self.screen, "--rotate", self.rotate])
+
+
+def instance():
```

Comment:
Let's even use `whereis_exe` in here as it might throw something similar to [this](https://ask.fedoraproject.org/en/question/98267/xrandr-doesnt-work-in-wayland-how-to-configure-display-resolution/) on wayland or basically on any other gnu/linux without Xrandr+X11.